language/move-lang/src/typing/expand.rs

```diff
@@ -257,10 +257,15 @@ fn bind(context: &mut Context, b: &mut T::Bind) {
     use T::Bind_ as B;
     match &mut b.value {
         B::Ignore => (),
-        B::Var(_, st_opt) => {
-            assert!(st_opt.is_some());
-            st_opt.iter_mut().for_each(|st| single_type(context, st))
+        B::Var(v, None) => {
+            let msg = format!(
+                "Unused local '{0}'. Consider removing or prefix with an underscore: '_{0}'",
```

Comment:
Is the `{0}` necessary as opposed to `{}`?

Also, what do you think of 'prefixing' as opposed to 'prefix'?

Reply:
Its used more than once, hence the `{0}`